api/src/main/resources/workbench-api.yaml

```diff
@@ -7787,3 +7814,29 @@ definitions:
           timestamp (epoch millis) at workspace creation
         type: integer
         format: int64
+  EditUserInformationRequest:
+    description: >
+      A group of changes an admin can make to a user in the Edit Information card of the individual-user admin page.
+      Updates to the contact email or the institutional affiliation will be rejected if they do not validate.
+    type: object
+    required:
+      - username
+    properties:
+      username:
+        description: The full system-assigned ID of a user, including email domain
+        type: string
+      freeCreditsLimit:
+        description: When set, overrides the default free credits dollar limit for this user.
+        type: number
+        format: double
+      contactEmail:
+        description: When set, replaces the user's contact email.  Must validate against the user's Institutional Affiliation, if one exists.
+        type: string
+      verifiedInstitutionalAffiliation:
+        description: When set, replaces the user's verified institutional email.  Must validate against the user's contact email.
```

Comment:
I'm confused: does the affiliation own the contact email or is it independent?

Reply:
copypaste error.  fixed.